src/parsers/manifestjson.js

```diff
@@ -623,6 +627,33 @@ export default class ManifestJSONParser extends JSONParser {
     if (this.parsedJSON.homepage_url) {
       this.validateHomePageURL(this.parsedJSON.homepage_url);
     }
+
+    this.validateRestrictedPermissions();
+  }
+
+  validateRestrictedPermissions() {
+    Object.keys(this.restrictedPermissions).forEach((permission) => {
+      const minVersion = this.restrictedPermissions[permission];
+
+      if (
+        (this.parsedJSON.permissions || [])
+          .map((perm) => perm.toLowerCase())
+          .includes(permission)
+      ) {
+        const { firefoxMinVersion: minVersionSetInManifest } =
+          this.getMetadata();
```

Comment:
Nit, we may do this once outside of the loop

Reply:
Yep